app/modules/assets/resources.py

```diff
@@ -130,9 +130,7 @@ def get(self, asset, format):
         except Exception:
             logging.exception('Got exception from get_or_make_format_path()')
             raise werkzeug.exceptions.NotImplemented
-        return send_file(
-            asset_format_path, 'image/jpeg'
-        )  # TODO we need to alter mime_type to reflect path, if ever it changes from jpg
+        return send_file(asset_format_path, asset.DERIVED_MIME_TYPE)
```

Comment:
thanks for busting this out into their own variables

Reply:
Do we always want the derived files to be jpegs?

Reply:
i suspect so, yes; at least for what we use assets for _now_.  namely:  they are usually just resized versions of some user-provided photographic content.   the only time i can _ever_ imagine us wanting to use a png would be if we cared about background transparency (e.g. a logo or something).   but i _think_ we have decided any such thing will be relegated to FileUpload at this point.  (so if we ever want to derive different formats of FileUploads -- this is likely something to consider differently.)   likewise if we let users upload animated gifs.   :grin: